src/next-app/components/Navbar.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import styles from '../styles/components/Navbar.module.css';
+import Button from './Button';
+import ProfileModal from './ProfileModal';
+import { IoMdMenu, IoMdContact } from 'react-icons/io';
+import Link from 'next/link';
+import { NavbarData } from './NavbarData';
+import { IconContext } from 'react-icons';
+
+function Navbar() {
+  const [sidebar, setSidebar] = useState(false);
+
+  const [profile, setProfile] = useState(false);
+
+  const showSidebar = () => setSidebar(!sidebar);
+
+  const showProfile = () => setProfile(!profile);
+
+  const logout = () => {
+    window.location.href = '/';
+  };
+
+  return (
+    <div>
+      <IconContext.Provider value={{ color: '#000' }}>
+        <div className={styles.navbar}>
+          <div className={sidebar ? styles['menu-bars-active'] : styles['menu-bars']}>
+            <IoMdMenu onClick={showSidebar} />
+          </div>
+          <div className={styles['btn-contianer']}>
+            <IoMdContact className={styles['profile-btn']} onClick={showProfile} />
+            {profile ? <ProfileModal /> : null}
+            <div className={styles['logout-btn']}>
+              <Button title={'Logout'} width={'120%'} onClick={logout} />
```

Comment:
Properties that are strings don't need curly braces around them. So this line should be `<Button title="Logout" width="120% onClick={logout}" />`